modules/violations-view-deployment-tab.adoc

```diff
@@ -11,34 +11,58 @@ The *Deployment* tab of the *Details* panel displays details of the deployment t
 [discrete]
 == Overview section
 
-The overview section lists the following information:
-
-* *Deployment ID*: The alphanumeric identifier for the deployment.
-* *Deployment name*: The name of the deployment.
-* *Deployment Type*: The type of the deployment.
-* *Cluster*: The name of the cluster where the container is deployed.
-* *Replicas*: The number of the replicated deployments.
-* *Namespace*: The unique identifier for the deployed cluster.
-* *Updated*: The time and date when the deployment was updated.
-* *Labels*: The labels that apply to the selected deployment.
-* *Annotations*: The annotations that apply to the selected deployment.
-* *Service Account*: The name of the service account for the selected deployment.
+The *Deployment overview* section lists the following information:
+
+* *Deployment ID*: The alphanumeric identifier for the deployment
+* *Deployment name*: The name of the deployment
+* *Deployment type*: The type of the deployment
+* *Cluster*: The name of the cluster where the container is deployed
+* *Namespace*: The unique identifier for the deployed cluster
+* *Replicas*: The number of the replicated deployments
+* *Created*: The time and date when the deployment was created
+* *Updated*: The time and date when the deployment was updated
+* *Labels*: The labels that apply to the selected deployment
+* *Annotations*: The annotations that apply to the selected deployment
+* *Service Account*: The name of the service account for the selected deployment
```

Comment:
All of the other lists have periods on the end of each entry, so suggest doing so here for consistency as well.